src/callbacks_step/stepsize_dg2d.jl

```diff
@@ -159,6 +159,22 @@ function max_dt(u, t, mesh::ParallelP4estMesh{2},
     return dt
 end
 
+function max_dt(u, t, mesh::ParallelT8codeMesh{2},
+                constant_speed::False, equations, dg::DG, cache)
+    # call the method accepting a general `mesh::T8codeMesh{2}`
+    # TODO: MPI, we should improve this; maybe we should dispatch on `u`
+    #       and create some MPI array type, overloading broadcasting and mapreduce etc.
+    #       Then, this specific array type should also work well with DiffEq etc.
+    dt = invoke(max_dt,
+                Tuple{typeof(u), typeof(t), T8codeMesh{2},
```

Comment:
Would it be possible to reuse the function from the p4est mesh? I.e. having a `mesh::Union{ParallelP4estMesh{2}, ParallelT8codeMesh{2}}` in the signature? Then, we would need a way to obtain the type `T8codeMesh{2}` from the `mesh`. `typeof(mesh)` probably returns `ParallelT8codeMesh{2}`, right? But is there another way maybe?

Reply:
Tried it.

Reply:
Actually we have
```
const ParallelT8codeMesh{NDIMS} = T8codeMesh{NDIMS, <:Real, <:True}
```
but see above.